### PR TITLE
Slider added

### PR DESCRIPTION
**Pull request by mridulnagpal**

### CODE REVIEW

**Comment by jywarren** [comment]
oh cool, does it resize dynamically? Could you add some space below it? I
think I could take a pass at styling it a bit more, but it's looking good
so far! I do think it may need a "scale" label?

**Comment by mridulnagpal** [comment]
@jywarren Yup its dynamic, my laptop was working slow. Added border and space. Added the scale label as well.

**Comment by jywarren** [comment]
This looks really cool. Do you want to remove the black box and make the "Scale" word bold like the other labels, then I can do a little styling work in CSS and we can merge?

**Comment by jywarren** [comment]
Also, i don't know, do you think left side should be most zoomed in?

**Comment by mridulnagpal** [comment]
@jywarren Made the changes, please have a look!!

**Comment by jywarren** [comment]
This looks great, but can we add:

`border-bottom: 2px solid #ccc;` - to the slider bar

also:
```

.slider::-webkit-slider-thumb {
    border-bottom: 2px solid #669;
}
```

And let's put the slider CSS into https://github.com/publiclab/leaflet-blurred-location/blob/master/dist/Leaflet.BlurredLocation.css ok? 

Also notice how this one has a tooltip that's nice! https://seiyria.com/bootstrap-slider/

Thanks Mridul!!! Looking really nice!

**Comment by mridulnagpal** [comment]
@jywarren Made the tweaks, working on bootstrap tooptip!!

**Comment by mridulnagpal** [comment]
@jywarren Done with the tooltip, please review!!

**Comment by mridulnagpal** [comment]
tooltip
Here is the demo!!

**Comment by mridulnagpal** [comment]
@jywarren should we merge this or add some more things?

**Comment by jywarren** [comment]
OK, so I pushed a few style changes to this branch, and it's looking good now. Can you add to the documentation that you need the extra includes, like the external stylesheet?

**Comment by jywarren** [comment]
Oh, but i also found that the tooltip wasn't showing the actual scale -- it kind of looks like it isn't in your gif either... maybe a small bug? 

But otherwise great, thanks!

**Comment by mridulnagpal** [comment]
@jywarren Made some changes, zoom doesn't change on each scale. I have logs before and after setZoom and both work fine. I don't understand what the problem is. Can you take a look?

**Comment by jywarren** [comment]
Sorry, what exactly is happening? Can you gif it?

**Comment by mridulnagpal** [comment]
bug
Here in console, the scale is before setZoom and "Done" is after. The code between the 2 is blurredLocation.setZoom(scale). And the method does take float inputs I checked. Any clue why this is happening?

**Comment by jywarren** [comment]
Is it possible that although it accepts floats, it will only actually "snap" to integer zoom levels? Is that what you're seeing?

**Comment by jywarren** [comment]
And if so, maybe this is ready for merge?

**Comment by mridulnagpal** [comment]
Yes, let's merge this, will open another issue for the bug.

File: examples/index.html [hunk 1]
```diff
@@ -103,12 +118,27 @@ <h3>Enter a location</h3>
 
       blurredLocation.panMapToGeocodedLocation("placenameInput");
 
-    </script>
+      var changeZoom = function() {
+        console.log(parseFloat(22 - parseFloat(slider.getValue()/10)));
+        blurredLocation.map.setZoom(parseFloat(22 - parseFloat(slider.getValue()/10)));
```
Comment (mridulnagpal) [inline examples/index.html]: #146 @jywarren This is the line which doesn't work sometimes,

**Comment by jywarren** [comment]
So, is the issue that, between integers, you're not seeing scale changes, and that it is only changing when it crosses an integer value? I think the method may /accept/ floats, but there may only exist map tiles for integer zoom levels, so it doesn't actually display anything between them?

**Comment by mridulnagpal** [comment]
@jywarren What's weird is that its not taking integer values it works on random values, that is why I am confused.

**Pull request merged**